hack/internal/tools/go.mod

```diff
@@ -1,6 +1,7 @@
 module sigs.k8s.io/kueue/internal/tools
 
-go 1.22.4
+go 1.22.6
```

Comment:
Please don't add this changes, we have separate PR for this https://github.com/kubernetes-sigs/kueue/pull/2859.